src/style.css

```diff
@@ -0,0 +1,116 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__img-box {
+  margin: 32px 19px 40px;
```

Comment:
all of your elements inside the card have the same margins to the  border of the card, so I think it's better to add padding to the whole card itself

Reply:
instead of adding margin to every element inside card container